test/conformance/tests/sriov_operator.go

```diff
@@ -905,101 +911,93 @@ var _ = Describe("[sriov] operator", func() {
 					}
 				})
 			})
-			Context("Resource Injector", func() {
-				// 25815
-				It("Should inject downward api volume", func() {
```

Comment:
Is there a reason why this was removed?
I think is a different test from those you are adding

Reply:
This is exactly the same test wrapped by TableTest wrapper. I just took existing tests transformed it to table test and add one more test "Network resource injector".